ws-to-kafka-app-deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: ws-to-kafka
+spec:
+  serviceName: ws-to-kafka
+  podManagementPolicy: Parallel
+  replicas: 2
```

Comment:
I think this will result in duplicate entries going into Kafka since both WebSocket consumers will be receiving the same data.

Reply:
sure, just testing scaling and probably forget to move it back to 1. Thank you